packages/bpk-component-barchart/hocs.js

```diff
@@ -31,13 +36,13 @@ export const withSelectedState = ComposedComponent => {
       };
     }
 
-    onBarClick = (e, { point }) => {
+    onBarClick = (e: SyntheticEvent<any>, { point }: { point: any }) => {
       this.setState({
         selectedPoint: point,
       });
     };
 
-    getBarSelection = point => isEqual(this.state.selectedPoint, point);
+    getBarSelection = (point: any) => isEqual(this.state.selectedPoint, point);
```

Comment:
We typically try to avoid `any` unless absolutely necessary, is there something more specific we can use here? We could also consider defining a `PointType` type, as I see this is repeated in a few places.

Reply:
Yeah I tried to find out what point actually is as a value but I couldn't find it used anywhere in the code and trying to type it as a string or number caused issues but happy to try again and get screenshots of what I mean